RELEASE_CHECKLIST.md

```diff
@@ -0,0 +1,14 @@
+# Release checklist
+
+This checklist should be worked through when releasing a new Serverless version.
+
+1. Update Segment.io key (never push the change to GitHub)
+2. Look through all open issues and PRs (if any) and close them / move them to another Milestone
+3. Look through all closed PRs and Issues to see what has changed
```

Comment:
Same here for the milestone right? There should also be some tools to create a changelog automatically for the future